src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random random = new Random();
+    private static final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public void getRandomBall() {
+        final int value = random.nextInt(100);
+        return new Ball(colorSupplier.getRandomColor(), value);
```

Comment:
void method can't return anything!
```suggestion
    public Ball getRandomBall() {
        final int value = random.nextInt(100);
        return new Ball(colorSupplier.getRandomColor(), value);
```